pom.xml

```diff
@@ -2920,18 +2920,40 @@
             <version>${pitest.plugin.version}</version>
             <configuration>
               <mutators>
-                <mutator>CONDITIONALS_BOUNDARY</mutator>
-                <mutator>CONSTRUCTOR_CALLS</mutator>
+                <mutator>REMOVE_CONDITIONALS_ORDER_IF</mutator>
+                <mutator>REMOVE_CONDITIONALS</mutator>
+                <mutator>REMOVE_CONDITIONALS_EQUAL_IF</mutator>
+                <mutator>TRUE_RETURNS</mutator>
+                <mutator>REMOVE_CONDITIONALS_EQUAL_ELSE</mutator>
+                <mutator>VOID_METHOD_CALLS</mutator>
+                <mutator>PRIMITIVE_RETURNS</mutator>
                 <mutator>FALSE_RETURNS</mutator>
-                <mutator>INCREMENTS</mutator>
+                <mutator>NON_VOID_METHOD_CALLS</mutator>
+                <mutator>EXPERIMENTAL_RETURN_VALUES_MUTATOR</mutator>
                 <mutator>INVERT_NEGS</mutator>
+                <mutator>CONDITIONALS_BOUNDARY</mutator>
+                <mutator>REMOVE_CONDITIONALS_ORDER_ELSE</mutator>
+                <mutator>DEFAULTS</mutator>
+                <mutator>RETURN_VALS</mutator>
+                <mutator>OLD_DEFAULTS</mutator>
+                <mutator>EXPERIMENTAL_SWITCH</mutator>
+                <mutator>RETURNS</mutator>
+                <mutator>EXPERIMENTAL_MEMBER_VARIABLE</mutator>
+                <mutator>NULL_RETURNS</mutator>
+                <mutator>EXPERIMENTAL_BIG_DECIMAL</mutator>
                 <mutator>MATH</mutator>
+                <mutator>EXPERIMENTAL_BIG_INTEGER</mutator>
+                <mutator>INCREMENTS</mutator>
+                <mutator>EXPERIMENTAL_ARGUMENT_PROPAGATION</mutator>
+                <mutator>EXPERIMENTAL_NAKED_RECEIVER</mutator>
```

Comment:
I am good with current list of mutators for this profile, including experimental mutators. In fact, I feel that these two experimental mutators in particular are very valuable. Here is an example of questionable code that `EXPERIMENTAL_ARGUMENT_PROPAGATION` caught:

```
    private static DetailNode getNearestNode(DetailNode node) {
        DetailNode tag = JavadocUtil.getNextSibling(node);
        // is replaced with 'DetailNode tag = node;'
        while (tag.getType() == JavadocTokenTypes.LEADING_ASTERISK
                || tag.getType() == JavadocTokenTypes.NEWLINE) {
            tag = JavadocUtil.getNextSibling(tag);
        }
        return tag;
    }

```